src/action.ts

```diff
@@ -42,7 +42,7 @@ export const getLatestVersion = async (): Promise<string | null> => {
     repo: 'terragrunt'
   });
 
-  return latestRelease.data.name;
+  return latestRelease.data.tag;
```

Comment:
There is no `tag` property in the response. `tag_name` needs to be used instead. Using `tag` results in the error:
```
Error: [tsl] ERROR in /home/runner/work/action-terragrunt/action-terragrunt/src/action.ts(45,29)
      TS2339: Property 'tag' does not exist on type '{ url: string; html_url: string; assets_url: string; upload_url: string; tarball_url: string | null; zipball_url: string | null; id: number; node_id: string; tag_name: string; target_commitish: string; ... 12 more ...; reactions?: { ...; } | undefined; }'.
```

```suggestion
  return latestRelease.data.tag_name;
```